### DISCUSSION TITLE
AXP2101 power out incorrect in firmware?

### DISCUSSION THREAD

**Discussion by faragher** [markqvist/Reticulum#482] [Help & Questions]
So, working with the AXP2101 and I have a question since it seems the schematics and the RNode firmware differ.

Output|Schematic|Firmware
---|---|---
ALDO2|GPS (3v3)|GPS(3v3)
ALDO3|LoRa (3v3)|LoRa (3v3)
DCDC1|3v3 Rail, ESP32|3v3 OLED
DCDC3|2v5 Rail OLED| 3v3 ESP32

Since both DCDC rails are at 3.3v it doesn't really matter, but am I reading something wrong? If someone plays with OLED intensity it could crash the MCU if it's on the wrong rail.

https://github.com/Xinyuan-LilyGO/LilyGo-LoRa-Series/blob/master/schematic/LilyGo_TBeam_V1.2.pdf

**Comment by faragher** [comment]
Okay, I'm pretty sure the schematic itself is wrong, since the voltage on the OLED 3.3v pin matches DCDC1. I'm really not impressed by the LilyGO documentation.

**Reply by markqvist** [reply]
I've consistently found discrepancies in their documentation and examples while working with it as well.
